server/services/notify/plugindelivery/subscription_deliver.go

```diff
@@ -53,7 +53,7 @@ func (pd *PluginDelivery) getDirectChannelID(teamID string, subscriberID string,
 			return "", fmt.Errorf("cannot find user: %w", err)
 		}
 		channel, err := pd.getDirectChannel(teamID, user.Id, botID)
-		if err != nil {
+		if err != nil || channel == nil {
 			return "", fmt.Errorf("cannot get direct channel: %w", err)
```

Comment:
Just reacting as I was navigating over there :P This would work with `err == nil` but the output message will look funky (probably something like) `cannot get direct channel: %!w(<nil>)`

Reply:
Yup - oops.

Reply:
I think I'll leave it.  This fix is only temporary (a day or two).